platform/platform_tests/local_country_file_tests.cpp

```diff
@@ -101,35 +101,65 @@ UNIT_TEST(LocalCountryFile_DiskFiles)
   CountryFile countryFile("TestCountry");
   countryFile.SetRemoteSizes(1 /* mapSize */, 2 /* routingSize */);
 
-  LocalCountryFile localFile(platform.WritableDir(), countryFile, 0 /* version */);
+  for (int64_t version : {0, 150312})
+  {
+    LocalCountryFile localFile(platform.WritableDir(), countryFile, version);
+    TEST(!localFile.OnDisk(MapOptions::Map), ());
+    TEST(!localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(!localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+
+    ScopedFile testMapFile(countryFile.GetNameWithExt(MapOptions::Map), "map");
+
+    localFile.SyncWithDisk();
+    TEST(localFile.OnDisk(MapOptions::Map), ());
+    TEST(!localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(!localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+
+    ScopedFile testRoutingFile(countryFile.GetNameWithExt(MapOptions::CarRouting), "routing");
+
+    localFile.SyncWithDisk();
+    TEST(localFile.OnDisk(MapOptions::Map), ());
+    TEST(localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+    TEST_EQUAL(7, localFile.GetSize(MapOptions::CarRouting), ());
+    TEST_EQUAL(10, localFile.GetSize(MapOptions::MapWithCarRouting), ());
+
+    localFile.DeleteFromDisk(MapOptions::MapWithCarRouting);
+    TEST(!testMapFile.Exists(), (testMapFile, "wasn't deleted by LocalCountryFile."));
+    testMapFile.Reset();
+
+    TEST(!testRoutingFile.Exists(), (testRoutingFile, "wasn't deleted by LocalCountryFile."));
+    testRoutingFile.Reset();
+  }
+}
+
+UNIT_TEST(LocalCountryFile_DiskFilesSingleMwm)
+{
+  Platform & platform = GetPlatform();
+
+  CountryFile countryFile("TestSingleMwmCountry");
+  countryFile.SetRemoteSizes(15 /* mapSize */, 1234 /* routingSize. It could have any value. */);
```

Comment:
Why not to set it to 0, instead of this magical 1234? And write a comment like "For 160707 we're assuming that routing sections are included into an mwm file, so there is no separate routing file at all."